Tools/Scripts/check-for-duplicated-platform-test-results

```diff
@@ -111,7 +106,7 @@ def find_duplicates_in_path(baseline_search_path):
         _log.debug(' comparing files in {0} and {1}'.format(remove_layout_test_path_prefix(remaining_paths[0]), remove_layout_test_path_prefix(remaining_paths[1])))
         for root, dirs, files in os.walk(remaining_paths[0]):
             for file_name in files:
-                if file_name in IGNORED_FILE_NAMES:
+                if '-expected' not in file_name:
```

Comment:
Pedantically, this probably is still unsafe—expectations can themselves have relative URLs in them, given reftest expectations can contain arbitrary content. That said, this seems like a strict progression?

Reply:
I would be in favor of banning tests with `-expected` in their name, to be honest. If nothing else for style reasons, even humans rely on a heuristic like this one.

Reply:
I meant the references of reftests, not the tests themselves—I think banning tests from having `-expected` in their name is sensible (and I think my old layout test finder rewrite branch may incidentally do that).